pyomo/gdp/plugins/bigm.py

```diff
@@ -19,29 +19,39 @@
     RangeSet)
 from pyomo.core.base import Transformation, TransformationFactory
 from pyomo.core.base.component import ComponentUID, ActiveComponent
+from pyomo.core.base.PyomoModel import ConcreteModel, AbstractModel
 from pyomo.core.kernel.component_map import ComponentMap
 from pyomo.core.kernel.component_set import ComponentSet
 from pyomo.gdp import Disjunct, Disjunction, GDP_Error
-from pyomo.gdp.util import target_list
+from pyomo.gdp.disjunct import _DisjunctData, SimpleDisjunct
+from pyomo.gdp.util import target_list, is_child_of
 from pyomo.gdp.plugins.gdp_var_mover import HACK_GDP_Disjunct_Reclassifier
 from pyomo.repn import generate_standard_repn
 from pyomo.common.config import ConfigBlock, ConfigValue
 from pyomo.common.modeling import unique_component_name
+from pyomo.common.deprecation import deprecation_warning
 from six import iterkeys, iteritems
+from weakref import ref as weakref_ref
 
 logger = logging.getLogger('pyomo.gdp.bigm')
 
 NAME_BUFFER = {}
 
 def _to_dict(val):
-    if val is None:
+    # [ESJ 09/14/2019] It doesn't seem like this can happen. Even if you
+    # explicitly specify it, this doesn't get called because None is the default
+    # value maybe?
+    # if val is None:
+    #     return val
```

Comment:
I think the result of this is that `None` becomes `{None: None}` instead of remaining `None`, no? Does that still give you the same result? I think so.

Reply:
Yes, you're right, it would be the same result anyway.